interceptors/realip/realip.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) The go-grpc-middleware Authors.
+// Licensed under the Apache License 2.0.
+
+package realip
+
+import (
+	"context"
+	"net"
+	"strings"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/peer"
+)
+
+type realipKey struct{}
+
+func FromContext(ctx context.Context) (net.IP, bool) {
```

Comment:
What do you think about supporting https://pkg.go.dev/net/netip#Addr? I know `peer.FromContext` returns a `net.IP` but it's sort of deprecated in favor of `netip.Addr`. It should be possible to use https://pkg.go.dev/net/netip#ParseAddr to parse from net.IP.

Reply:
I've refactored to use netip.Addr ;)